### ISSUE TITLE
Discussion: Suggestion to establish 9:16 as default aspect ratio for Web Stories (including in the Desktop viewer)

### ISSUE THREAD

**Issue by pbakaus** [ampproject/wg-stories#28]
We've started discussing this on an internal email thread, but moving it to a public forum now where all WG members can engage.

**Tl;DR**: I'm working on an AMP story editor for WordPress, and we'd like to propose to change the default aspect ratio for mobile stories to **9:16**, which is particularly relevant for when consuming mobile stories on Desktop. Our team also recommends this aspect ratio to other story editors, unless you support responsive story creation. You'll find a lot of background and some prototypes to play with in [this explainer doc](https://docs.google.com/document/d/1eoa7dncGNrv9JT_Z2ANVvEp_2FesI9f2hnozCSWK2l0/edit).

A couple arguments I made in favor (taken slightly out of context, happy to explain in more detail):

- Most video is shot 9:16, so most video will be cut off or letterboxed, regardless of what we will do in the editor
- If we don't do this, it will mean that every story of our editor will be letterboxed (at least the content area, the background might fill or not fill based on setting).
- Current percentage-based story editors actually break on lots of phones (I confirmed this with a large partner). Unless they're doing something magical, this will continue until they follow suit and letterbox.
- We will letterbox all stories, in one way or another (some with background that fills, luckily). It's not a matter of whether we want to or not – it's very hard to build a magic layout self-healing responsive system that adapts to what the user intended, without exposing responsive design tools. @dvoytenko  and I spend weeks researching, building prototypes.
- I still want that crazy magical system, but it's many months away from reality, unless somebody in our team has a breakthrough idea we haven't considered before.

Would love to hear arguments before or against,

**Comment by cpauwels** [comment]
@pbakaus @dvoytenko The way we here at Tappable solved responsive design in a way that makes it easy for users is to use a grid. In our case, we made it 24x40. This provides a lot of flexibility to lay out text, images and videos while making it very feasible to make stories responsive. You just need to have a cover/contain option for images and videos, with the cover having a way to set a focus point, and contain an option to align and center.

Here is a story that we built that way: [https://stories.tappable.dev/brusselsbeerproject](https://stories.tappable.dev/brusselsbeerproject?story). It's responsive all the way from square to taller than 1:2 without effort or knowledge needed from the user.

We do understand the need to create a fixed aspect ratio layer, it would also be nice to have for layouts that don't handle responsive design very well. But I would strongly reconsider the 9/16 AR. As device and browser chrome is virtually always present, even with tall (<= 1/2) devices, you virtually never reach up to a 9/16 aspect ratio. As an example, the tall iPhones (no home button) have a Safari viewport AR equal to 0.59, which is way closer to 3/5 (0.6) than 9/16 (0.5625) We at Tappable thus chose 3/5 (same as the amp-story-player library) as the default aspect ratio. 

This only minimally crops out 9/16 videos while showing significantly more content from 16/9 videos (which let's be honest, is still the overwhelming majority of non-user-generated content). Here are a few figures

*Content shot in 16/9:*
Viewed 9/16: 31% visible
Viewed 3/5: 37% visible (+19%)

*Content shot in 9/16*
Viewed 9/16: 100% visible
Viewed 3/5: 93% visible (-7%)

And finally, I have a few questions about how you would go about handling AMP UI elements such as swipe up and CTAs that are positioned not related to the layer but the viewport. Do they sort-of overlap the fixed AR layer depending on the viewport size or are they contrained within?

**Comment by dvoytenko** [comment]
Thanks, @cpauwels! This is very useful. But to clarify: we are talking singularly only about a more confined change: the default aspect ratio for the "desktop panels" mode. This mode is currently 3:5 and is already aimed at stories with somewhat less responsive design. Our main reason for change: a 9:16 video that's sized to cover the 3:5 ratio would lose about 11% of the vertical video space. We've been finding that the vertical space loss is especially bad given when it contains subtitles or other text info.

**Comment by pbakaus** [comment]
I'd like to maybe separate what is right for editors to what is right for the Desktop preview mode.

I think what is right for editors is a longer conversation – @cpauwels is right that when optimizing for the most popular device resolution, which is 9:16 (with ~43% market share according to public data from DeviceAtlas and StatCounter), the stories will most commonly viewed at something like 9:13 or 9:14 if taking the browser UI into account.

But that doesn't change the fact that we anticipate video content shot in 9/16 to be one of the most common backgrounds on story pages, and if we don't want to crop any content, we would decide to use that resolution for the Desktop mobile preview mode. Editors like @cpauwels that can deal with responsiveness should have no trouble adapting, and I don't think we should optimize for 16/9 video as background element (these videos should generally always be reformatted in pre-production, or on the fly with some video CDN).

Paging @cramforce @newmuis and @jasti for thoughts.

**Comment by dvoytenko** [comment]
Let's close this, since we decided not to change it.
